source/math/Quaternion.ooc

```diff
@@ -33,7 +33,7 @@ Quaternion: cover {
 	// z = this y
 	// w = this z
 
-	inverse ::= This new(this w, -this x, -this y, -this z)
+	inverse ::= This new(this w / (this norm * this norm), -this x / (this norm * this norm), -this y / (this norm * this norm), - this z / (this norm * this norm))
```

Comment:
I'm not suggesting we keep the incorrect implementation at all &mdash; I'm asking that we *first* make sure that any code that is dependent on the current implementation of `inverse` calls `conjugate` and *then* correct the implementation of `inverse`.